test/katello_test_helper.rb

```diff
@@ -43,6 +43,8 @@ def load_permissions
   end
 end
 
+ActiveSupport::Deprecation.silenced = true
```

Comment:
@ehelms I removed this line and changed ```ActiveRecord::Fixture``` to ```ActiveRecord::FixtureSet``` and changed the classes from strings to constants in fixtures_helper.rb which got rid of any deprecation warnings